packages/ketcher-core/jest.config.js

```diff
@@ -10,5 +10,12 @@ module.exports = {
     'domain(.*)$': '<rootDir>/src/domain/$1',
     'infrastructure(.*)$': '<rootDir>/src/infrastructure/$1',
     'utilities(.*)$': '<rootDir>/src/utilities/$1'
+  },
+  globals: {
```

Comment:
if we add a global variable in ketcher-core, do we need the same in other packages?

Reply:
It's not a common variable. It's a config for ts-lint. It allows running test with TS warning. Otherwise, tests fail.